fl4health/preprocessing/autoencoders/loss.py

```diff
@@ -0,0 +1,73 @@
+from typing import Tuple
+
+import torch
+from torch.nn.modules.loss import _Loss
+
+
+class VAE_loss(_Loss):
```

Comment:
For classes, I believe the typical code style is to use camel case with no underscores. With acronyms only starting with a capital letter, this class would be `VaeLoss`. The use of the `_` in `_Loss` is an exception to indicate internal use for the package (i.e. it's exclusively a parent class).